packages/web-app/src/components/common/Properties/Duration.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
+
+const DurationWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  & > span {
+    margin-left: 5px;
+  }
+`;
+
+const Duration = ({ image, durationStr, title }) => {
+  const { formatMessage } = useIntl();
+  const splittedTime = durationStr.split(':');
+  const valueToDisplay = `${
+    +splittedTime[0] > 0 ? `${+splittedTime[0]}h ` : ''
+  }${+splittedTime[1]}m`;
```

Comment:
Dans `useIntl` il existe `formatTime` pour ce genre d'opérations. On ne l'a pas encore trop utilisé mais je pense qu'il faut que ça devienne la norme dans l'appli.

Reply:
@Clm-Roig : Bien noté pour useIntl, cependant dans ce cas je dois formaté dans un format "duration" qui n'est ni une date ni un time...
J'ai regardé la doc et je n'ai pas l'impression que l'on puisse faire une durée en h et minutes avec useIntl ...